src/components/Captures/CaptureTable.js

```diff
@@ -206,17 +215,29 @@ const CaptureTable = () => {
   };
 
   return (
-    <Grid style={{ height: '100%', overflow: 'auto' }}>
+    <Grid style={{ height: '100%', overflow: 'auto', padding: '8px 32px' }}>
       <Grid
         container
         direction="row"
         justify="space-between"
         alignItems="center"
       >
-        <Typography variant="h5" className={classes.title}>
-          Captures
+        <Typography variant="h5">
+          {captureCount !== null &&
+            `${countToLocaleString(captureCount)} capture${
+              captureCount === 1 ? '' : 's'
```

Comment:
I don't think the `!== null` is required. You can simplify the check to
` {captureCount && ... }`

Reply:
Not quite. `captureCount === 0` is a valid value that would fail `{captureCount && ...}`

Reply:
I'll add a comment there to clarify